OrbitClientModel/CaptureSerializer.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientModel/CaptureSerializer.h"
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <google/protobuf/message.h>
+
+#include <fstream>
+#include <memory>
+
+#include "Callstack.h"
+#include "EventTracer.h"
+#include "FunctionUtils.h"
+#include "OrbitBase/MakeUniqueForOverwrite.h"
+#include "OrbitProcess.h"
+#include "absl/strings/str_format.h"
+#include "capture_data.pb.h"
+
+using orbit_client_protos::CallstackEvent;
+using orbit_client_protos::CallstackInfo;
+using orbit_client_protos::CaptureInfo;
+using orbit_client_protos::FunctionInfo;
+using orbit_client_protos::FunctionStats;
+using orbit_client_protos::TimerInfo;
+
+void CaptureSerializer::WriteMessage(const google::protobuf::Message* message,
+                                     google::protobuf::io::CodedOutputStream* output) {
+  uint32_t message_size = message->ByteSizeLong();
```

Comment:
`ByteSizeLong()` returns a `size_t`. So this is usally 64 bit wide. 

`uint64_t` and `WriteLittleEndian64 `would probably have been a better choice. I'm not 100% sure about changing it. My wild guess is that we don't handle captures >4GB great currently (so we don't really win anything). 
On the other hand captures >4GB might be a thing in the long run and the earlier we fix this the easier it is.

Reply:
For clarification:
- AFAIK protobuf has a maximum message size of 2 gigs.
- A serialized capture is a stream of protos, meaning `WriteMessage` will only encode a single timer at once which should not be more than a couple of bytes.

Reply:
Thanks for that clarification!

Reply:
As discussed above and in the standup - this is not a problem. Should stay as is.

Reply:
Discussed offline, I will leave it as it is. Thanks!